src/app/workflow/workflow.component.html

```diff
@@ -47,6 +47,26 @@ <h2 class="text-center">The workflow linked to could not be found!</h2>
       This {{ workflow.entryTypeMetadata.term }} was archived on {{ workflow.dbUpdateDate | date: 'yyyy-MM-dd' }}. It is now read-only.
     </mat-card>
   </div>
+  <div class="row m-1 mt-3 mb-0" *ngIf="!isWorkflowPublic && workflow.mode === 'DOCKSTORE_YML' && gitHubAppInstalled === false">
+    <mat-card class="alert alert-warning mat-elevation-z m-0">
+      This {{ workflow.entryTypeMetadata.term }} may not be automatically updating because the Dockstore GitHub App was uninstalled from the
+      {{ workflow.entryTypeMetadata.term }}'s repository. If you are the repository owner, click the button below to reinstall the Dockstore
```

Comment:
minor nitpick:
suggest `a owner or admin of the repository` instead of  `the repository owner` to hint that more roles can do this than one user (full explanation is probably too complicated) (at https://docs.github.com/en/apps/using-github-apps/installing-a-github-app-from-a-third-party#requirements-to-install-a-github-app )